db/app.js

```diff
@@ -1,23 +1,36 @@
 const express = require("express");
 const app = express();
+app.use(express.json());
 
 const {
   getAllTopics,
   getAllEndpoints,
   getArticleById,
-  getAllArticles
+  getAllArticles,
+  getCommentForArticle,
+  addCommentById
 } = require("../controllers/nc_news.controller");
 
 app.get("/api", getAllEndpoints);
 app.get("/api/topics", getAllTopics);
 app.get("/api/articles/:article_id", getArticleById);
 app.get("/api/articles", getAllArticles);
+app.get("/api/articles/:article_id/comments", getCommentForArticle)
+app.post("/api/articles/:article_id/comments", addCommentById)
 
 app.all("/*", (req, res, next) => {
   res.status(404).send({ msg: "Not found" });
   next(err);
 });
 
+app.use((err, req, res, next) => {
+  if (err.code === "23503") {
```

Comment:
You might want to look to soon abstract your error handling into an error file :)